recipe/meta.yaml

```diff
@@ -16,26 +16,41 @@ build:
 
 requirements:
   host:
-    - python >=3.6
+    - python >=3.8,<3.12
     - pip
   run:
-    - python >=3.6
-    - tiktoken
-    - dataclasses-json
-    - langchain >=0.0.303
-    - sqlalchemy >=2.0.15
-    - numpy
-    - tenacity >=8.2.0,<9.0.0
-    - openai >=0.26.4
-    - pandas
-    - urllib3 <2
-    - fsspec >=2023.5.0
-    - typing_inspect >=0.8.0
-    - typing-extensions >=4.5.0
-    - beautifulsoup4
-    - nest-asyncio
-    - nltk
-    - tree_sitter_languages
+    - python >=3.8<=3.12
+    - SQLAlchemy ={extras = ["asyncio"], version = ">=1.4.49"}
```

Comment:
this syntax is not supported in conda recipes

try something like

```
    - sqlalchemy >=1.4.49
    - greenlet !=0.4.17
```

see https://github.com/sqlalchemy/sqlalchemy/blob/44771ab0c32730d1ecefac21af13f113b1b19726/setup.cfg#L43